bokehjs/src/coffee/models/sources/columnar_data_source.ts

```diff
@@ -32,6 +34,10 @@ export abstract class ColumnarDataSource extends DataSource {
   streaming: Signal<any, this>
   patching: Signal<any, this> // <number[], ColumnarDataSource>
 
+  inspected: Selection
+  selection_policy: SelectionPolicy
+  selection_manager: SelectionManager
```

Comment:
This is incorrect. All all model attribute declarations must go into `Attrs` interfaces, otherwise you won't be able to fully construct this model (on the type-level obviously). Consult the original merge commit, because there may be more of this.